### ISSUE TITLE
MatrixRankWarning for angle section with thickness equal to toe radius

### ISSUE THREAD

**Issue by Ernst79** [robbievanleeuwen/section-properties#421]
- **sectionproperties version**: 3.2.1
- **Python version**: 3.11
- **OS version and name**: Windows

- [x] I am on the [latest](https://github.com/robbievanleeuwen/section-properties/releases/latest)
      stable sectionproperties version, installed using a recommended method.
- [x] I have searched the [issues](https://github.com/robbievanleeuwen/section-properties/issues)
      of this repo and believe that this is not a duplicate.
- [x] I have consulted the [documentation](https://sectionproperties.readthedocs.io/)
      for any relevant information.

## Issue

First of all, thanks for this wonderful piece of work. 

I have a small issue, when creating and analyzing an angle section. 

```python
from sectionproperties.pre.library import angle_section
from sectionproperties.analysis import Section

geom = angle_section(d=20, b=20, t=3, r_r=3.5, r_t=3, n_r=16)
geom.create_mesh(mesh_sizes=[20 / 20])

# create mesh
sec = Section(geometry=geom)

# analyse mesh
sec.calculate_geometric_properties()
sec.calculate_warping_properties()
sec.calculate_plastic_properties()

ixx_c, iyy_c, ixy_c = sec.get_ic()
zxx_plus, zxx_minus, zyy_plus, zyy_minus = sec.get_z()

print("area:", int(sec.get_area()))
print("Ixx:", int(ixx_c))
print("Iyy:", int(iyy_c))
print("Ixy:", int(ixy_c))
```

The above code return the following warning

![image](https://github.com/robbievanleeuwen/section-properties/assets/24375847/683908ae-33dc-40ca-b019-c16cb8720971)

The problem only occurs if the toe radius is the same as the thickness of the angle, probably due to the vertical part of the bottom flange is getting a length of t - r_t = 0.00000.

**Comment by robbievanleeuwen** [comment]
Hi @Ernst79, thanks for submitting the issue. This has been fixed with #429. There will be a new release shortly that incorporates this fix!

**Comment by Ernst79** [comment]
Thanks!
